From: Markus Koschany <apo@debian.org>
Date: Mon, 4 Dec 2017 01:52:25 +0100
Subject: remove deprecated gnome-common macros

---
 autogen.sh | 2 --
 1 file changed, 2 deletions(-)

diff --git a/autogen.sh b/autogen.sh
index f026881..1594066 100755
--- a/autogen.sh
+++ b/autogen.sh
@@ -20,6 +20,4 @@ which gnome-autogen.sh || {
     exit 1
 }
 REQUIRED_AUTOMAKE_VERSION=1.8 \
-USE_GNOME2_MACROS=1 \
-USE_COMMON_DOC_BUILD=yes \
 . gnome-autogen.sh
